dll/win32/fmifs/query.c

```diff
@@ -43,3 +49,135 @@ QueryAvailableFileSystemFormat(
 
     return TRUE;
 }
+
+/**
+ * @brief
+ * Retrieves removable disk device information, mostly related to Sony Memory Stick devices.
+ * On Vista+ the returned information is disk sector size and volume length in sectors,
+ * regardless of the type of disk.
+ *
+ * @param[in] DriveRoot
+ * String which contains a DOS device name,
+ *
+ * @param[in,out] DeviceInformation
+ * Pointer to buffer with DEVICE_INFORMATION structure which will receive data.
+ *
+ * @param[in] BufferSize
+ * Size of buffer in bytes.
+ */
```

Comment:
For the sake of documentation you can add function return information here (returns TRUE on success).